.husky/pre-commit

```diff
@@ -2,5 +2,3 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npm run security-checks
-
-npx prettier --ignore-unknown --write $(git diff --name-only --staged | tr '\n' ' ') | grep -o '^\S*' | xargs -I {} git add {}
```

Comment:
why was this removed?

Reply:
Need to revert these, tried bringing it up to date with develop but the merge messed up, 'cause my node hadn't started.